src/screens/TestCases/TestDraft/TestDraftDetails.js

```diff
@@ -190,30 +122,55 @@ function TestDraftDetails(props) {
     setSelectedFilters([]);
   };
 
-  // const handleApplyFilter = () => {
-  //   let newData = getDraftTestListData.filter(
-  //     (project) => project.is_active === 1
-  //   );
-  //   if (selectedFilters.length > 0 && filterColumn) {
-  //     newData = newData.filter((row) =>
-  //       selectedFilters.includes(row[filterColumn])
-  //     );
-  //   }
-  //   setFilteredData(newData);
-  //   closeModal();
-  // };
-
-  const handleApplyFilter = () => {
-    let newData = getDraftTestListData;
-    if (selectedFilters.length > 0 && filterColumn) {
-      newData = getDraftTestListData.filter((row) =>
-        selectedFilters.includes(row[filterColumn])
+  const [filterType, setFilterType] = useState('');
```

Comment:
make it one state convert into use reducer hooks

Reply:
resolved